asto-redis/src/test/java/com/artipie/asto/redis/RedisStorageTest.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.redis;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.amihaiemil.eoyaml.YamlMapping;
+import com.artipie.asto.ArtipieIOException;
+import com.artipie.asto.Content;
+import com.artipie.asto.Key;
+import com.artipie.asto.Meta;
+import com.artipie.asto.Storage;
+import com.artipie.asto.blocking.BlockingStorage;
+import com.artipie.asto.ext.ContentAs;
+import com.artipie.asto.factory.Storages;
+import io.reactivex.Single;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.redisson.Redisson;
+import org.redisson.config.Config;
+import org.testcontainers.containers.GenericContainer;
+
+/**
+ * Tests for {@link RedisStorage}.
+ *
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle MagicNumberCheck (500 lines)
+ * @since 0.1
+ */
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.TooManyMethods"})
+@DisabledOnOs(OS.WINDOWS)
+public final class RedisStorageTest {
```

Comment:
can we add `RadisStorage` into `StorageExtension` to test this storage as we do test all the storages?

Reply:
created an [issue](https://github.com/artipie/asto/issues/445)